src/index.html

```diff
@@ -7,8 +7,33 @@
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <title>Moyo header</title>
     <link rel="stylesheet" href="./style.css">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com"
+    value="crossorigin">
```

Comment:
let's formatting this line
```suggestion
    <link
      rel="preconnect"
      href="https://fonts.gstatic.com"
      value="crossorigin"
    >
```

<img width="269" alt="Снимок экрана 2023-09-05 в 16 12 03" src="https://github.com/mate-academy/layout_moyo-header/assets/75898310/dfa07901-0298-4b90-8dbc-efc6de90b535">